### PR TITLE
Instructions on how to install github-gem on Windows (v2)

### PR DESCRIPTION
**Pull request by jwillmer**
I already created a [pull request](https://github.com/jekyll/jekyll/pull/4975) for this topic but it was asked to include the instructions instead of linking them so I did that in this pull request. I also included the instructions for the installation of Jekyll.

### CODE REVIEW

File: site/_docs/windows.md [hunk 1]
```diff
@@ -44,3 +46,74 @@ with Windows. Add the following to the Gemfile for your site:
...
+ gem install nokogiri --^
+   --with-xml2-include=C:\Chocolatey\lib\libxml2.2.7.8.7\build\native\include^
+   --with-xml2-lib=C:\Chocolatey\lib\libxml2.redist.2.7.8.7\build\native\bin\v110\x64\Release\dynamic\cdecl^
+   --with-iconv-include=C:\Chocolatey\lib\libiconv.1.14.0.11\build\native\include^
+   --with-iconv-lib=C:\Chocolatey\lib\libiconv.redist.1.14.0.11\build\native\bin\v110\x64\Release\dynamic\cdecl^
+   --with-xslt-include=C:\Chocolatey\lib\libxslt.1.1.28.0\build\native\include^
+   --with-xslt-lib=C:\Chocolatey\lib\libxslt.redist.1.1.28.0\build\native\bin\v110\x64\Release\dynamic
+```
+
+#### Install github-gem
```
Comment (parkr) [inline site/_docs/windows.md]: The gem is technically called `github-pages` – could you update all your references to use this consistent name?

**Comment by parkr** [comment]
This is great! More than happy to add this, too.

**Comment by parkr** [comment]
@jekyll/windows Is anyone able to help confirm this?

**Comment by jwillmer** [comment]
can someone explain why the test failed after I renamed some names?!

File: site/_docs/windows.md [hunk 2]
```diff
@@ -10,12 +10,14 @@ knowledge and lessons that have been unearthed by Windows users.
 
 ## Installation
 
-Julian Thilo has written up instructions to get
-[Jekyll running on Windows][windows-installation] and it seems to work for most
-people. The instructions were written for Ruby 2.0.0, but should work for later
-versions [prior to 2.2][hitimes-issue].
+A quick way to install Jekyll it to follow the [installation instructions by David Burela](https://davidburela.wordpress.com/2015/11/28/easily-install-jekyll-on-windows-with-3-command-prompt-entries-and-chocolatey/):
```
Comment (ayastreb) [inline site/_docs/windows.md]: Should it be "to install Jekyll **is** to follow" instead of "to install Jekyll **it** to follow" ?

**Comment by parkr** [comment]
Thank you for this!

@jekyllbot: merge +site

**Pull request merged**